generator/config/schema.json

```
@@ -58,7 +58,7 @@
                     "type": "string",
                     "enum": [
                         "array",
-                        "object",
+                        "flat_object",
```

`object` is still used.
```suggestion
                        "object",
                        "flat_object",
```

---

Whoops. Done.